src/ImageList/index.js

```diff
@@ -172,9 +172,15 @@ export default class ImageList extends Component {
 
     if (!items) {
       if (hasUpdatedLoadingStates) {
-        let height = columnCount === 2 ? _height / 2 : _height
+        // TODO (michael-adalo): flag is now set to ON for all environments,
+        // we could remove the check and default to the logic below
```

Comment:
I found this comment quite confusing, but I guess the fix is setting the `onLayout` prop?

Reply:
Yup. The comment relates to the use of the hasUpdatedLoadingStates flag. I think we can remove it in a future update.